src/megaclient.cpp

```diff
@@ -4260,9 +4237,16 @@ void MegaClient::removeCaches()
         }
     });
 
-    // TODO: dgw: truncate?
-    if (!mKeepSyncsAfterLogout)
+    if (keepSyncsConfigFile)
     {
+        // Special case backward compatibility for MEGAsync
+        // The syncs will be disabled, if the user logs back in they can then manually re-enable.
+        syncs.disableSyncs(LOGGED_OUT, false);
+    }
+    else
+    {
+        // TODO: actually delete file?
```

Comment:
`if (!keepSyncsConfigFile)`, then yes, we should actually delete the file.

Reply:
In my tests it does delete the file. We should remove the TODO

Reply:
removed